bin/docker-entrypoint

```diff
@@ -60,26 +60,29 @@ workers_healthcheck() {
 
 server() {
   # Recycle gunicorn workers every n-th request. See http://docs.gunicorn.org/en/stable/settings.html#max-requests for more details.
+
+  MAX_REQUESTS=${MAX_REQUESTS:-1000}
+  MAX_REQUESTS_JITTER=${MAX_REQUESTS_JITTER:-100}
+  TIMEOUT=${REDASH_GUNICORN_TIMEOUT:-60}
+
+  args=()
   case $REDASH_PRODUCTION in
     true)
       echo "Starting Redash Server in production mode"
-      MAX_REQUESTS=${MAX_REQUESTS:-1000}
-      MAX_REQUESTS_JITTER=${MAX_REQUESTS_JITTER:-100}
-      TIMEOUT=${REDASH_GUNICORN_TIMEOUT:-60}
-      exec /usr/local/bin/gunicorn \
-        -b 0.0.0.0:5000 \
-        --name redash \
-        -w${REDASH_WEB_WORKERS:-4} redash.wsgi:app \
-        --max-requests $MAX_REQUESTS \
-        --max-requests-jitter $MAX_REQUESTS_JITTER \
-        --timeout $TIMEOUT
       ;;
     *)
       echo "Starting Redash Server in a dev mode"
       export FLASK_DEBUG=1
-      exec /app/manage.py runserver --debugger --reload -h 0.0.0.0
+      args+=("--reload")
       ;;
   esac
+  exec /usr/local/bin/gunicorn \
+    -b 0.0.0.0:5000 \
+    --name redash \
+    -w${REDASH_WEB_WORKERS:-4} redash.wsgi:app \
+    --max-requests $MAX_REQUESTS \
+    --max-requests-jitter $MAX_REQUESTS_JITTER \
+    --timeout $TIMEOUT ${args[@]}
```

Comment:
Let's make "$args" a simple string instead of a BASH array

Reply:
moved `--reload` arg to compose file instead and added ability to pass command extra args to server process, which can be useful for helm chart as well